### PR TITLE
Story/524 Add helper function to simulate edi cron calls

### PR DESCRIPTION
**Pull request by PeterAlabaster**
Needed to prevent race conditions caused by dispatching to cron multiple times in quick succession

Signed-off-by: Peter Alabaster <peter.alabaster@unipart.io>

User-story: 524

### CODE REVIEW

**Review by PeterC10 — APPROVED** [review]

File: addons/edi/tests/common.py [hunk 1]
```diff
@@ -188,3 +188,17 @@ def assertRaisesIssue(self, entity, exception=UserError):
 
         # Delete the raised issue
         new_issue_ids.unlink()
+
+    def _sim_edi_cron_execution(self, document_type):
```
Comment (ws-mps) [inline addons/edi/tests/common.py]: This function can be used for other tests not simulating crons, so I would suggest to rename it to something more generic, and we probably do not need references to crons.

**Review by armandcela — APPROVED** [review]

File: addons/edi/tests/common.py (hunk 1 above)
Comment (PeterAlabaster) [reply addons/edi/tests/common.py]: something like this?
```python
def _execute_edi_for(self, document_type):
   ...

# Usage
self._execute_edi_for(self.document_type_abc)
```

Comment (PeterAlabaster) [reply addons/edi/tests/common.py]: i'm never too hot on naming things, especially generic functions

Comment (ws-mps) [reply addons/edi/tests/common.py]: That sounds good to me. I'm not great with names either :)

**Review by ws-mps — APPROVED** [review]

**Pull request merged**